### PR TITLE
fix: Possible to provide an escape hatch for expressions

### PR DESCRIPTION
**Pull request by wugeer**
fix: https://github.com/shssoichiro/sqlformat-rs/issues/15

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,8 @@ categories = ["development-tools"]
 [dependencies]
 nom = "7.0.0"
 unicode_categories = "0.1.1"
+lazy_static = "1"
```
Comment (shssoichiro) [inline Cargo.toml:18]: The most recent recommendation is to migrate from `lazy_static` to [`once_cell`](https://github.com/matklad/once_cell)

**Review by shssoichiro — CHANGES_REQUESTED** [review]
Thanks, this looks really good overall, just one change recommended.

File: Cargo.toml (hunk 1 above)
Comment (wugeer) [reply Cargo.toml]: @shssoichiro Done! :)

**Review by shssoichiro — APPROVED** [review]
Thanks!

**Comment by max-sixty** [comment]
Awesome, thanks a lot!

**Pull request merged**